src/main/resources/org/jenkinsci/plugins/workflow/libs/ResourceStep/help-libraryName.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    The shared library name to use to locate the resource.
+    If left blank, all loaded shared libraries will be loaded.
```

Comment:
nit: Maybe reword to "If left blank, then this step will search for the resource in all configured shared libraries."